.gitignore

```diff
@@ -7,3 +7,4 @@
 .env.*
 release/
 scripts/
+.idea/
```

Comment:
You should ignore IDE specific files and folders in your own global gitignore.

Reply:
ok, fixed.